test/parallel/test-assert-deep.js

```diff
@@ -157,28 +157,26 @@ assert.throws(
 // but deepStrictEqual should throw.
 {
   const similar = new Set([
-    { 0: '1' },  // Object
     { 0: 1 },  // Object
     new String('1'),  // Object
-    ['1'],  // Array
     [1],  // Array
     date2,  // Date with this[0] = '1'
     re2,  // RegExp with this[0] = '1'
     new Int8Array([1]), // Int8Array
-    new Uint8Array([1]), // Uint8Array
+    // TODO: Fix this check...
+    // new Uint8Array([1]), // Uint8Array
```

Comment:
It looks like this check is commented out on purpose, perhaps for a later PR, but I just wanted to double check if that is the case?

Reply:
`Buffer` is an `Uint8Array` and without checking the prototype, this is identical. I guess just removing the entry would be best.